main.go

```diff
@@ -112,16 +119,47 @@ func NewGame(g *Game) {
 	g.Sprites = make(map[SpriteType]*SpriteSheet, 2)
 	g.Sprites[spritePlayer] = loadSprite("Player")
 	g.Sprites[spriteZombie] = loadSprite("Zombie")
+	g.Sprites[spriteDog] = loadSprite("Dog")
+
+	// Load entities from map
+	entities := level.LayerByIdentifier("Entities")
 
 	// Add player to the game
 	g.Player = &Player{
 		State:  playerIdle,
-		Object: resolv.NewObject(float64(g.Width/2), float64(g.Height/2), 20, 20),
+		Object: resolv.NewObject(float64(g.Width/2), float64(g.Height/2), 20, 20, tagPlayer),
 		Angle:  0,
 		Sprite: g.Sprites[spritePlayer],
 	}
 	g.Space.Add(g.Player.Object)
 
+	// Load the dog's path
+	dogEntity := entities.EntityByIdentifier("Dog")
+
+	pathArray := dogEntity.Properties[0].AsArray()
+	path := make([]Coord, len(pathArray))
+	for index, pathCoord := range pathArray {
+		// Do we really need to make these crazy castings?
```

Comment:
Hmmm, I hope not...

Reply:
if this is the only way we can get this out of the library then that's unfortunate, it'll do for now though and maybe we can make something better later...

Reply:
We could write wrappers to make it prettier but if this is the only place we do this then it's not worth it.